examples/node/lib/rest/api/V2010.ts

```diff
@@ -14,25 +14,25 @@
 
 import ApiBase from '../ApiBase';
 import Version from '../../base/Version';
-import { AccountListInstance } from './v2010/accounts';
+import { AccountListInstance } from './v2010/account';
 
 export default class V2010 extends Version {
     /**
      * Initialize the V2010 version of Api
      *
-     * @property { Twilio.Api.V2010.AccountListInstance } accounts - accounts resource
+     * @property { Twilio.Api.V2010.AccountListInstance } account - account resource
      *
      * @param { Twilio.Api } domain - The Twilio domain
      */
     constructor(domain: ApiBase) {
         super(domain, 'v2010');
     }
 
-    protected _accounts?: AccountListInstance;
+    protected _account?: AccountListInstance;
```

Comment:
Shouldn't this be plural?

https://github.com/twilio/twilio-node/blob/f120b4b011c4f5d25c5acf47bc23679e52799353/lib/rest/api/V2010.d.ts#L48

Reply:
It should, but mount names are currently derived from the ancestry in the resource tree (which automatically singularizes every resource). Will need to spend some time refactoring the `ResourceMap` such that we can still access non-singularized resource names